src/lesson3/LessonThree.java

```diff
@@ -0,0 +1,155 @@
+package lesson3;
+
+import java.util.Scanner;
+
+public class LessonThree {
+    public static void main(String[] args) {
+        Scanner scanner = new Scanner(System.in);
+
+        // 1. Multiplication Table
+        getMultiplicationTable(scanner);
+
+        // 2. Every Seventh Number
+        getEverySeventhNumber(scanner);
+
+        // 3. Sequence of Numbers
+        getSequenceOfNumbers(scanner);
+
+        // 4. Population Size
+        getPopulationSize();
+
+        // 5-6. Final Deposit Sum
+        getFinalDeposit(scanner);
+
+        // 7-8. Day of Week
+        getDayOfWeek(scanner);
+
+        // 9. Factorial
+        getFactorial(scanner);
+    }
+
+    public static void getMultiplicationTable(Scanner scanner) {
+        int attempt = 0;
+        while (attempt != 3) {
+            System.out.print("Enter a number: ");
+            int number = scanner.nextInt();
+            if (number > 9 || number < 1) {
+                System.out.println("Enter from 1 to 9.");
+            } else {
+                for (int i = 1; i <= 10; i++) {
+                    System.out.println(number + "*" + i + "=" + number * i);
+                }
+                break;
+            }
+            attempt++;
+        }
+    }
+
+    public static void getEverySeventhNumber(Scanner scanner) {
+        System.out.print("Enter a number: ");
+        int number = scanner.nextInt();
+        for (int i = 1; i <= number; i++) {
+            if (i % 7 == 0) {
+                System.out.print(i + " ");
+            }
+        }
+    }
+
+    public static void getSequenceOfNumbers(Scanner scanner) {
+        System.out.print("Enter a length of a sequence: ");
+        int length = scanner.nextInt();
+        for (int i = 1, b = 1; i <= length; i++, b *= 2) {
+            System.out.print(b + " ");
+        }
+    }
```

Comment:
Змінна i тут наче не портібна
```suggestion
        for (int b = 1; b <= length; b *= 2) {
            System.out.print(b + " ");
        }
    }
```

Reply:
але тоді не виконається умова  🤔

Reply:
якщо я вкажу довжину послідовності 9, то замість 1 2 4 8 16 32 64 128 256, отримаю 1 2 4 8

Reply:
ну так, можна замість length  вказати 512, і тоді буде відповідати умові. Або залишай як є